cinder/volume/drivers/lightos.py

```diff
@@ -766,7 +766,7 @@ def _delete_lightos_volume(self, project_name, lightos_uuid):
             if status_code == httpstatus.OK:
                 break
 
-            LOG.info(
+            LOG.warn(
```

Comment:
not critical, but suugest to add to returned string "retrying operation.
i.e:
"delete_volume for volume with LightOS UUID %s failed \
￼                with status code %s response %s, retrying operation"
so it is clearer why this is a warning and not error.

also note, post merge,   not sure if we want need this retry, as we also have a retry loop within send_cmd,
additionally lightOS may return error not_found. in which case it is redundent to retry error.

we need to look at the retry logic further, also possibly in lightOS need to make delete volume idempotent (i.e if voule already deleted no error)